src/jaegertracing/LogRecord.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 Uber Technologies, Inc.
+ * Copyright (c) 2017-2018 Uber Technologies, Inc.
```

Comment:
I'm sorry to be annoying about this. The idea was to only change the modified files. That way we only update the changed files. See this commit for a related change in the main Jaeger repo: https://github.com/jaegertracing/jaeger/commit/d7d1c74fa74f77d2c7faa50a0c568f979d9a30fd.

Reply:
Is this what you're looking for? The files I added have 2018 and the file that got modified has 2017-2018.